teuthology/task/internal/__init__.py

```diff
@@ -318,7 +318,11 @@ def fetch_binaries_for_coredumps(path, remote):
             # 1422917770.7450.core: ELF 64-bit LSB core file x86-64, version 1 (SYSV), SVR4-style, \
             # from 'radosgw --rgw-socket-path /home/ubuntu/cephtest/apache/tmp.client.0/fastcgi_soc'
             log.info(f' core looks like: {dump_out}')
-            dump_program = dump_out.split("from '")[1].split(' ')[0]
+            try:
+                dump_program = dump_out.split("from '")[1].split(' ')[0]
+            except:
+                log.info("core doesn't have the desired format, moving on ...")
```

Comment:
Should we go ahead and do something like `log.debug(f"dump_out=")` as well?

Reply:
wouldn't be the same as ` log.info(f' core looks like: {dump_out}')` in the line above? or do you mean changing log.info to log.debug

Reply:
oh I just plain didn't see that line - disregard 😄